src/OrbitGl/App.cpp

```diff
@@ -2060,8 +2060,8 @@ void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
                                  "/org/freedesktop/FileManager1",
                                  "org.freedesktop.FileManager1.ShowItems",
                                  QString::fromStdString(absl::StrFormat(
-                                     "array:string:\"file:///%s\"", preset.file_path().string())),
```

Comment:
`"array:string:\"file:///%s\""` is fine when I directly type in the command in the terminal. But when running with `QProcess::startDetached`, this format string will result in an additional `"` at the end of the file path.

Reply:
That's expected because the command is not running through a shell. Try this: `"array:string:file:///%s"`

Reply:
Ah. I just saw that you had already changed it. But in your version there are 4 slashes: `file:////` Theoretically 2 should be enough when `%s` is an absolute path.

Reply:
I tried two slashes, but it does not work... 4 works for me

Reply:
In that case I guess we just leave the 4 slashes. Thank you, Vicky! :-)